server/controllers/middlewares/verifyUser.js

```diff
@@ -0,0 +1,26 @@
+const { verify } = require('jsonwebtoken');
+
+const verifyArtist = async (req, res, next) => {
+  const decoded = await verify(req.cookies.token, process.env.SECRET_KEY);
```

Comment:
`Jwt.verify` is an async callback, not a built-in promise so, you can't use `async/await` with it, and if we want to use promise with it we need to build it using `new promise(reject, resolve)`, or you can directly deal with it using callback as I mentioned in suggesting.
```suggestion
 verify(req.cookies.token, process.env.SECRET_KEY, (err, payload) => {
```